apps/fabric-website/package.json

```diff
@@ -18,34 +18,28 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@types/chai": "3.4.35",
-    "@types/enzyme": "2.8.8",
-    "@types/es6-promise": "^0.0.32",
-    "@types/mocha": "2.2.39",
+    "@types/es6-promise": "0.0.32",
     "@types/node": "8.0.26",
-    "@types/prop-types": "15.5.1",
+    "@types/prop-types": "15.5.2",
     "@types/resemblejs": "~1.3.28",
-    "@types/sinon": "2.2.2",
-    "@types/react": "15.0.38",
-    "@types/react-dom": "15.5.1",
+    "@types/react": "16.0.25",
+    "@types/react-dom": "16.0.3",
```

Comment:
I am thinking maybe we should close this PR, and reopen one that has the change you intended :)